lib/dom.js

```diff
@@ -327,13 +345,25 @@ declare class HTMLElement extends Element {
     id: string;
     title: string;
     lang: string;
+    translate: boolean;
     dir: string;
     className: string;
+    accessKey: string;
+    accessKeyLabel: string;
+    dataset: {[key:string]: string};
+    contentEditable: string;
+    isContentEditable: boolean;
+    hidden: boolean;
+    tabIndex: number;
+    spellcheck: boolean;
+    getBoundingClientRect(): ClientRect;
+    insertAdjacentHTML(position: string, text: string): void;
```

Comment:
[Looks like this belongs on `Element`](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML)